linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -129,6 +129,15 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_LINUX_07_010: [** `insert_idx` and `consume_idx` for the task array shall be initialized to 0. **]**
 
+[Will be moved from threadpool_open]
+**S_R_S_THREADPOOL_LINUX_11_001: [** `threadpool_create` shall initialize internal threapool data items **]**
+
+[Will be moved from threadpool_open]
+**S_R_S_THREADPOOL_LINUX_07_020: [** `threadpool_create` shall create number of `min_thread_count` threads for `threadpool` using `ThreadAPI_Create`. **]**
```

Comment:
>create number [](http://example.com/codeflow?start=63&length=13)

create *a* number.

Reply:
Actually scratch that, it should be 
create `min_thread_count` threads

Reply:
Yeah, grammar issue

Reply:
Done